packages/dynamic_forms/lib/src/form_elements/property_names.dart

```diff
@@ -0,0 +1,9 @@
+final String VALUE_PROPERTY_NAME = "value";
```

Comment:
It is a good idea to have file with common property names, but it makes it hard to rename one property on a single FormElement. Also it would make it difficult to generate FormElement code automatically in the future. I would probably rather move it inside the form elements itself even though it will be a lot of duplication. It can even stay public, because right now when consuming forms on the client we need to send propertyName argument to the method, which will change the state of the form if user inserts some value, so we would just use this constant on a specific FormElement.